.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -3,18 +3,18 @@ Release checklist:
 Tagging:
  - [ ] Write release notes in NEWS. Get them reviewed and merged
      - [ ] If doing a branched release, also include a PR to merge the NEWS changes into main
- - [ ] Ensure your local copy is up to date with the upstream main branch (`git@github.com:coreos/fcct.git`)
+ - [ ] Ensure your local copy is up to date with the upstream main branch (`git@github.com:coreos/butane.git`)
  - [ ] Ensure your working directory is clean (`git clean -fdx`)
  - [ ] Ensure you can sign commits and any yubikeys/smartcards are plugged in
  - [ ] Run `./tag_release.sh <vX.Y.z> <git commit hash>`
  - [ ] Push that tag to GitHub
 
 Packaging:
- - [ ] Update the FCCT spec file in [Fedora](https://src.fedoraproject.org/rpms/fedora-coreos-config-transpiler):
+ - [ ] Update the Butane spec file in [Fedora](https://src.fedoraproject.org/rpms/butane):
```

Comment:
Renaming dist-git is a separate step, no? I'm actually not sure what the procedure for that is. For this checklist, should we just leave the previous location for now?

Reply:
It is a separate step, and in fact the package rename policy requires a re-review.  I'm planning to do that before the next release, so I figured I'd go ahead and update the procedure.  If we don't get dist-git renamed in time, I don't think this change will cause any substantial confusion.